ui/src/track-layout/layout-design-api.ts

```diff
@@ -16,8 +16,19 @@ export type LayoutDesign = {
     id: LayoutDesignId;
 } & LayoutDesignSaveRequest;
 
-export const getLayoutDesigns = async (changeTime: TimeStamp) =>
-    designCache.get(changeTime, '', () => getNonNull<LayoutDesign[]>(`${baseUri}/`));
+export const getLayoutDesigns = async (
+    includeDeleted: boolean,
+    includeCompleted: boolean,
```

Comment:
Nykymallilla näistä kannattaisi varmaankin tehdä jonkinlaisia enumeja booleanien sijaan. Boolean-flagit on kuitenkin se malli mitä meillä on käytössä about kaikissa muissakin listausrajapinnoissa, joten tietoisesti päätin käyttää sitä sen sijasta että olisi tehnyt tänne jonkun enumin. Jos näitä halutaan joskus siirtää enumimalliin, niin voisi harkita jonkinlaista kaikki rajapinnat kerralla refaktoroivaa yhteisyliheittoa.